library/src/main/java/com/airbnb/android/airmapview/GoogleChinaWebViewMapFragment.java

```diff
@@ -1,9 +1,12 @@
 package com.airbnb.android.airmapview;
 
 public class GoogleChinaWebViewMapFragment extends WebViewMapFragment {
-
   public static GoogleChinaWebViewMapFragment newInstance(AirMapType mapType) {
     return (GoogleChinaWebViewMapFragment) new GoogleChinaWebViewMapFragment()
         .setArguments(mapType);
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
but The china webview fragment should be the same as teh google one but with a different URL. So at the very least we should be able to copy the solution from the GoogleWebViewMapFragment here.